Sources/PrimerSDK/Classes/Data Models/PaymentMethodConfigurationOptions.swift

```diff
@@ -66,5 +66,43 @@ struct AsyncPaymentMethodOptions: PaymentMethodOptions {
     
 }
 
+struct BlikPaymentMethodOptions: PaymentMethodOptions {
+    
+    let paymentMethodType: PaymentMethodConfigType
+    let paymentMethodConfigId: String
+    let type: String = "OFF_SESSION_PAYMENT"
+    let sessionInfo: SessionInfo?
+    
+    private enum CodingKeys : String, CodingKey {
+        case type, paymentMethodType, paymentMethodConfigId, sessionInfo
+    }
+    
+    init(
+        paymentMethodType: PaymentMethodConfigType,
+        paymentMethodConfigId: String,
+        sessionInfo: SessionInfo?
+    ) {
+        self.paymentMethodType = paymentMethodType
+        self.paymentMethodConfigId = paymentMethodConfigId
+        self.sessionInfo = sessionInfo
+    }
+    
+    func encode(to encoder: Encoder) throws {
+        var container = encoder.container(keyedBy: CodingKeys.self)
+        try container.encode(type, forKey: .type)
+        try container.encode(paymentMethodType.rawValue, forKey: .paymentMethodType)
+        try container.encode(paymentMethodConfigId, forKey: .paymentMethodConfigId)
+        try? container.encode(sessionInfo, forKey: .sessionInfo)
+    }
+    
+    struct SessionInfo: Codable {
+        let blikCode: String
+        let locale: String
+        let platform: String = "IOS"
```

Comment:
I believe in another PR we are reviewing to avoid "warnings". Then we will update with [this ticket](https://primerapi.atlassian.net/browse/DEX-1291)